src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const thead = table.querySelector('thead tr');
+const tbody = table.querySelector('tbody');
+const rows = tbody.querySelectorAll('tr');
+const number = string => string.replace(/[$,]/g, '');
+
+thead.addEventListener('click', even => {
+  const headerIndex = Array.from(thead.children).indexOf(even.target);
+
+  if (even.target.innerHTML === 'Salary') {
+    Array.from(rows)
+      .sort((a, b) => {
+        const aValue = +number(a.children[headerIndex].textContent);
+        const bValue = +number(b.children[headerIndex].textContent);
+
+        return aValue - bValue;
+      }).forEach(row => {
+        tbody.appendChild(row);
+      });
+  } else {
+    Array.from(rows)
+      .sort((a, b) => {
+        const aValue = a.children[headerIndex].textContent;
+        const bValue = b.children[headerIndex].textContent;
+
+        return aValue.localeCompare(bValue);
+      }).forEach(row => {
+        tbody.appendChild(row);
```

Comment:
Actually, there is no need to have two totally different logics for salary and other values, as many moments are the same for all columns. Just provide only different sorting if it's the salary column